quickstart/201-vm-disk-encryption-extension/outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "admin_ssh_key" {
+  sensitive = true
+  value     = tls_private_key.vm_key.private_key_pem
```

Comment:
Could we output this private key via a [`local_file` resource](https://registry.terraform.io/providers/hashicorp/local/latest/docs/resources/file)? And we'd better include `local` provider's version restriction in `terraform` block in `providres.tf` file.